metadata.yaml

```diff
@@ -14,4 +14,7 @@ releaseSeries:
     contract: v1alpha3
   - major: 0
     minor: 3
-    contract: v1alpha3
\ No newline at end of file
+    contract: v1alpha3
+  - major: 0
```

Comment:
Should we still be including the v1alpha3 refs if we no longer support it?

Reply:
I'm not really sure. I just made this match other providers in the way they set this.

Reply:
Gotcha.  They offer backward-compatible v1alpha3/4 APIs, though.  If/since we're no longer offering those API versions, I'd rather we just make like they never existed.

Reply:
Okay, I'll just delete the v1alpha3 stuffs with early versionings.

Reply:
to be clear, this commit itself won't serve as a version bump/release, right? We'll keep developing against it before we cut a final tag/release?

Reply:
Yes, for sure. : )